purchase_fop_shipping/README.rst

```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License AGPL-3
+
+===========================
+Pruchase Free-Of-Paiment shipping
+===========================
+
+This module allows to define a Free-Of-Paiment (FOP) shipping on supplier.
+FOP shipping is a min order amount to got shipping free from supplier.
+User can force confirm order if he/she wishes. 
+
+Installation
+============
+
+You need an Odoo/OCB installation v 10.0.
+
+Usage
+=====
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/purchase-workflow/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/purchase-workflow/issues/new?body=module:%20purchase_fop_shipping%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+
+Credits
+=======
+
+Contributors
+------------
+
+* Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com.br>
+
+Icon
+----
+
+* Original Odoo purchase module icon
+* https://openclipart.org/detail/159937/clothing-label-with-rope
+
+Maintainer
+----------
+
+.. image:: http://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: http://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```

Comment:
could you use the last template ?
https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst
and about == title, you have to have the same number of '=' as the number of char in the tilte.